src/styles/_variables.scss

```diff
@@ -38,4 +40,7 @@ $secondary-hover: #682C73;
 $warning-hover: #D98E25;
 $alert-hover: #D93B30;
 
-$third: rgb(130,153,156);
\ No newline at end of file
+$third: rgb(130,153,156);
```

Comment:
if you want to go with the naming convention of primary, secondary, etc... it should be tertiary. Otherwise, a more generic name for the colors use case it more suitable